<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewMyAttendance</class>
 <widget class="QDialog" name="viewMyAttendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>671</height>
    </rect>
   </property>
   <widget class="QLabel" name="background1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y2:0, x2:1, y2:1, stop:0 rgba(85, 98, 112, 255), stop:1 rgba(255, 107, 107, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>631</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#fff;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>View Attendance</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRollNo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Enter your Roll No :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/new/assets/assets/degreehat.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rollView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>175</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,200);
color:rgb(0,0,0);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Roll No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectYear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Select Year :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="yearView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	color:black;
	background-color:white;
	border: 1px solid #ced4da;
	border-radius:4px;
	padding-left: 10px;
}
QComboBox::drop-down{
	border:0px;
}
QComboBox::down-arrow{
	image: url(:/new/assets/assets/downArrow.png);
	width:12px;
	height:12px;
	margin-right: 10px;
}
QComboBox:on{
	border:2px solid red;
}
QComboBox QListView{
	font-size:12px;
	border:1px solid rgba(0,0,0,10%);
	padding:5px;
	background-color: #fff;
	outline: 0px;
}
QComboBox QListView::item{
	padding-left: 10px;
	background-color: #fff;
}
QComboBox QListView::item:hover{
	background-color: #D2122E;
}
QComboBox QListView::item:selected{
	background-color: #D2122E;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SE Comp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TE Comp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblSelectSubject">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Select Subject :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subjectView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>315</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	color:black;
	background-color:white;
	border: 1px solid #ced4da;
	border-radius:4px;
	padding-left: 10px;
}
QComboBox::drop-down{
	border:0px;
}
QComboBox::down-arrow{
	image: url(:/new/assets/assets/downArrow.png);
	width:12px;
	height:12px;
	margin-right: 10px;
}
QComboBox:on{
	border:2px solid red;
}
QComboBox QListView{
	font-size:12px;
	border:1px solid rgba(0,0,0,10%);
	padding:5px;
	background-color: #fff;
	outline: 0px;
}
QComboBox QListView::item{
	padding-left: 10px;
	background-color: #fff;
}
QComboBox QListView::item:hover{
	background-color: #D2122E;
}
QComboBox QListView::item:selected{
	background-color: #D2122E;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color:rgba(85,98,112,255);
	color:rgba(255,255,255,200);
	border-radius:5px;
}
QPushButton:pressed
{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255,107,107,255);
	background-position:calc(100% -10px)center;
}
QPushButton:hover
{
	background-color:rgba(255,107,107,255);
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/new/assets/assets/degreehat.png</normaloff>:/new/assets/assets/degreehat.png</iconset>
    </property>
   </widget>
   <zorder>background2</zorder>
   <zorder>background1</zorder>
   <zorder>lblHeader</zorder>
   <zorder>lblRollNo</zorder>
   <zorder>lblLogo</zorder>
   <zorder>rollView</zorder>
   <zorder>lblSelectYear</zorder>
   <zorder>yearView</zorder>
   <zorder>lblSelectSubject</zorder>
   <zorder>subjectView</zorder>
   <zorder>pushButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
